docs/release-notes/2.35.0.md

```diff
@@ -0,0 +1,106 @@
+---
+name: Percona Monitoring and Management <version>
+about: Release Notes Template for PMM
+
+---
+
+| **Release date:** | [Month DD, YYYY]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+
+## Release Highlights
+
+###  Access Control: Limit access to metrics 
+!!! caution alert alert-warning "Important/Caution"
+    Disclaimer:  PMM Access Control is still technical preview and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+We are excited to announce the first step towards PMM Access Management, a highly anticipated solution by our users. PMM now allows you to limit access to metrics, based on labels. This new feature provides a standardized method for granting, changing, and revoking access to metrics. You can now establish policies that control which data can be queried based on specific label criteria with the introduction of label-based access control. It allows you to associate multiple sets of individual Prometheus (Victoria Metics) labels with a policy, ensuring that queries will only return data from series that meet at least one of the defined labels. This provides you with a powerful and flexible way to manage your data and ensure secure access to the information you need. For more information, please visit the Access Control page. 
```

Comment:
"Prometheus (Victoria Metics) labels"
We should avoid using technical language in Highlights, as this supports "oh, it's Prom + Grafana ...."

Reply:
ok